src/main/java/frc/robot/Constants.java

```diff
@@ -133,6 +133,18 @@ public static class NoteDetectionConstants {
         public static double screenHeightPixels;
         
     }
+
+    public static class JoystickConstants {
+        public static final double deadZoneRange = 0.15;
+        public static final double deadZoneRotation = 0.10;
+        
+        public static final double maxLinearSpeedms = 4.5;
+        public static final double maxRotationalSpeedDegrees = 180;
+
+        public static final double limitedOutput = 0.8;
```

Comment:
id recommend changing this variable name to something more descriptive like joystick linearity, and adding a comment line indicating what values do what because the units are not clear. (0 fully linear, 1 fully exponential for example).

Reply:
Agreed.  Done.